functionality.js

```diff
@@ -0,0 +1,18 @@
+function toggle(id) {
+    const sidebarData = document.getElementById(id);
+    if (sidebarData.style.display === 'none') {
```

Comment:
instead of if, you can use ternary to practice ES6

Reply:
done